### DISCUSSION TITLE
Debian 10+: apt install flameshot isn't connected to v.12

### DISCUSSION THREAD

**Discussion by elad2109** [flameshot-org/flameshot#2747] [Q&A]
![image](https://user-images.githubusercontent.com/1093841/176976454-49bcd6fd-e477-4091-9d5c-8a20afbe50ca.png)

**Comment by mmahmoudian** [comment]
In the following link you can see the latest available version of Flameshot in most of the linux distro official repositories:

https://repology.org/project/flameshot/versions

If any of them are outdated other than AUR, there is nothing we can do, and you should contact the maintainer of flameshot in that distro.
